test/integration/framework/src/siftool/common.py

```diff
@@ -108,6 +108,7 @@ def basic_logging_setup():
     logging.getLogger("eth").setLevel(logging.WARNING)
```

Comment:
This part sets the logging levels. We set the overall logging level to `DEBUG`, but since certain external libraries such as web3 emit a lot of noise with `DEBUG`, we silence those libraries to `WARNING` level.